maintainers/google-season-of-docs-2024.md

```diff
@@ -0,0 +1,84 @@
+- Beta test the new documentation, fix sharp edges and add potential missing information about the most asked used cases. For this we will use our forum and social media presence to find all level of nix users and ask them to try the new docs.
+- Respond to the communities feedback and work on all the identified blind spots
+- Add release note entries, write announcement posts and finally publish the new documentation
+- Delete the old documentation and link to the new one where appropriate
+
+Work that is out-of-scope for this project:
+
+- We will not refactor documentation bits that aren't directly related to the mentioned topics
+- We will not add automatic documentation tests since that is a very large project on its own and missing in many more places than the mentioned topics
+- Working on version specific docs is out of scope since the API's for said topics are stable
+
+We already approached some potential technical writing candidates, but none of them are able to commit until we have a confirmed budget.
+As mentioned in the budgeting plan below, we are looking for two candidates, one per topic. One of the candidates for the docker documentation is @adamcstephens on GitHub, he has a lot of experience with container technologies and already know's his way around nixpkgs and related tooling. As for volunteer's @Janik-Haag would be willing to work on upcoming tasks, including the ones from the budget item justification.
+
+Measuring the project's success:
+
+Nix has a repository called nixpkgs which is our official package collection.
+It includes packages such as libraries and programs, but also nix helper for building software.
+Every package in this repo gets build by our ci and the build artifacts are uploaded to our binary cache (https://cache.nixos.org).
+If someone now was to cross compile something using nix tooling, nix would go and fetch the needed builder from the binary cache.
+We measure the number of cache hits per entry using our content delivery network provider.
+
+We would consider the cross compilation documentation successful if cache hits for said builder have a noticeable up tick in relation to overall nixpkgs usage.
+
+Measuring the success of docker build tools is a bit more tricky since everything nix needs to build docker images is basically gnu-coreutils and python with it's stdlib.
+This means a rise in cache hits for python and coreutils as result of better dockerTools documentation would basically be indistinguishable from for example a large company adopting nix as their primary build tool.
+What we can do instead is plot a chart of https://github.com/search?q=lang%3Anix+dockerTools&type=code and compare if the dockerTools usage goes up in relation to overall nixpkgs usage.
+
+We would consider the docker documentation successful if there is an exponential dockerTools usage increase compared to before and with relation to general nixpkgs usage.
```

Comment:
```suggestion
Measuring the success of improving documentation for `dockerTools` is a bit more tricky, since Nix uses no special dependencies to build Docker images.
This means a rise in cache hits for Python and Coreutils as result of better `dockerTools` documentation would be indistinguishable from a large company adopting Nix as their primary build tool.
What we can do instead is plot a chart of [`dockerTools` usage](https://github.com/search?q=lang%3Anix+dockerTools&type=code) and check if it goes up in relation to overall Nixpkgs usage.

We would consider the docker documentation successful if
- There is an substantial increase in use of `dockerTools` compared to before and  relation to general Nixpkgs usage.
- Traffic on the `dockerTools` guide increases significantly
```